library/ts-rollup/README.md

```diff
@@ -1,16 +1,41 @@
-# ♻️ Usage
+# Environment with TypeScript and Rollup
+
+## ♻️ Usage
+
+The production bundle is available as [ESM bundle](./dist/bingo-generator.esm.js).
+
+### Initialize the library
+```javascript
+    import { Bingo } from 'bingo-generator';
+    
+    // Initialize bingo-generator
+    // 'bingo-container' is the id of the HTMLElement that renders the Bingo
+    const bingo = new Bingo('bingo-container');
+```
+
+### Generate a new grid with random numbers on each square
+```javascript
+    bingo.resetGrid();
+```
+The result should be something like this:
+  <p align="center">
+    <img title="Grid" src="../../images/grid.png">
+  </p>
+
+## Technical part
 
 The project is configured with Jest for the tests, Playwright & Jest-image-snapshot for the visual tests, ESLint for the code formatting.
 
-## IDE Configuration Tips
+### IDE Configuration Tips
 
-### IntelliJ/WebStorm
+#### IntelliJ/WebStorm
```

Comment:
⚠️ same, too much changes to backport, I have just change both README recently and this is a pain (cf #11)